mobify/platform-scaffold
# Stub out header footer

So we have a structure similar to the PWA

~~**JIRA**: (link to JIRA ticket)~~

## Changes
- Add Header stub
- Add Footer stub
- Add App skeleton container

## How to test-drive this PR
- Run `npm run dev`
- Observe the DOM
